src/modules/zoom/zoom.mjs

```diff
@@ -378,6 +377,7 @@ export default function Zoom({ swiper, extendParams, on, emit }) {
   }
   function onTouchEnd() {
     const zoom = swiper.zoom;
+    evCache.length = 0;
```

Comment:
On Chrome for iOS, long pressing a slide's image would leave the previous touch event in the `evCache` array.
This ensure that `evCache` is cleared `onTouchEnd`.